lib/js_browser.mli

```diff
@@ -17,6 +17,61 @@ module Storage : sig
   val clear: t -> unit
 end
 
+module Date : sig
+  type t = private Ojs.t
+  val t_of_js: Ojs.t -> t
+  val t_to_js: t -> Ojs.t
+
+  val new_date: int -> t [@@js.new]
+  val now : unit -> int [@@js.global "Date.now"]
```

Comment:
Sorry I'm a bit confused. What I understand from [`js_of_ocaml` manual](https://ocsigen.org/js_of_ocaml/2.8.4/manual/library) is that integers are represented by the [`Number` class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) so there should be no conversion involved and thus no risk of overflow, right?